src/monitors/longnotready.js

```diff
@@ -96,8 +96,8 @@ class PodLongNotReady extends EventEmitter {
 	}
 
 	checkRecovery(item, readyStatus) {
-		if (this.alerted[`${item.metadata.namespace}/${pod.metadata.name}`] && config.get('recovery_alert')) {
-			delete this.alerted[`${item.metadata.namespace}/${pod.metadata.name}`];
+		if (this.alerted[`${item.metadata.namespace}/${item.metadata.name}`] && config.get('recovery_alert')) {
```

Comment:
I'll need to PR that fix again, this is the correct version -.-

Reply:
Haha. I've been working to fix that in master